src/libs/tmdk/src/commands/init.ts

```diff
@@ -17,51 +14,57 @@ import path from "path";
 import {
   ListComponentTypesCommandOutput,
   ListScenesCommandOutput,
+  ListEntitiesCommandOutput,
+  GetEntityCommandOutput,
 } from "@aws-sdk/client-iottwinmaker";
 
 export type Options = {
-  region: string | undefined;
-  "workspace-id": string | undefined;
-  out: string | undefined;
+  region: string;
+  "workspace-id": string;
+  out: string;
+};
+
+export type tmdk_config_file = {
+  version: string;
+  component_types: string[];
+  scenes: string[];
+  models: string[];
+  entities: string;
 };
 
 export const command = "init";
 export const desc = "Initializes a tmdk application";
 
-export const builder: CommandBuilder<Options, Options> = (yargs) =>
+export const builder: CommandBuilder<Options> = (yargs) =>
   yargs.options({
     region: {
       type: "string",
-      require: false,
+      require: true,
       description:
         "Specify the AWS region of the Workspace to bootstrap the project from.",
       defaultDescription: "$AWS_DEFAULT_REGION",
       default: process.env.AWS_DEFAULT_REGION,
```

Comment:
Nit: If the option is required can we remove the default values?

Reply:
will update

Reply:
Also applies to other commands, but not a priority to remove